docker/BASEIMAGE

```diff
@@ -4,3 +4,4 @@ windows/amd64/1809=mcr.microsoft.com/windows/nanoserver:1809
 windows/amd64/1903=mcr.microsoft.com/windows/nanoserver:1903
 windows/amd64/1909=mcr.microsoft.com/windows/nanoserver:1909
 windows/amd64/2004=mcr.microsoft.com/windows/nanoserver:2004
+windows/amd64/2004=mcr.microsoft.com/windows/nanoserver:ltsc2022
```

Comment:
nit: please add newline.

Earlier there were _two newlines_ and only one was expected to remove. EOF is expected to have a _newline_.